stats_test.go

```diff
@@ -165,6 +174,23 @@ func TestDistributionData(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
+			data := &view.Data{
+				View: newView(view.Distribution(tc.Bounds...)),
+				Rows: []*view.Row{
+					{
+						Tags: []tag.Tag{},
+						Data: &view.DistributionData{
+							CountPerBucket:  []int64{0, 2, 3},
+							Min:             1,
+							Max:             5,
+							Mean:            3,
+							SumOfSquaredDev: 10,
+							Count:           15,
+						},
+					},
+				},
+			}
+
```

Comment:
Here too. The curly braces already create enough spacing.

Reply:
Sure, fixed.